src/main/io/serial.c

```diff
@@ -156,17 +162,24 @@ const char* serialPortNames[SERIAL_PORT_COUNT] = {
 #endif
 };
 
-const uint32_t baudRates[] = {0, 9600, 19200, 38400, 57600, 115200, 230400, 250000,
-        400000, 460800, 500000, 921600, 1000000, 1500000, 2000000, 2470000}; // see baudRate_e
+const uint32_t baudRates[BAUD_COUNT] = {
```

Comment:
(the idea was different, but don;t worry about it, it is unlikely that new baudrate will be added. 
It's style that is quite resilient to refactoring mistakes: )
```suggestion
const uint32_t baudRates[] = {
 [BAUD_AUTO] = 0, [BAUD_9600] = 9600, .. 
 };
 // extern const uint32_t baudRates[]; in serial.h is necessary for assert to work
 STATIC_ASSERT(ARRAYLEN(baudRates) == BAUD_COUNT. "baudRates does not match enum");
```